js/checkout.js

```diff
@@ -180,6 +200,76 @@ class Blockonomics {
     redirect_to_finish_order() {
         window.location.href = this.data.finish_order_url
     }
+
+    _set_refresh_loading(loading=false) {
+        if(loading) {
+            this._refresh.querySelector('.blockonomics-icon-refresh').classList.add('spin')
+            this._refresh.setAttribute('disabled', 'disabled')
+        } else {
+            this._refresh.querySelector('.blockonomics-icon-refresh').classList.remove('spin')
+            this._refresh.removeAttribute('disabled')
+        }
+    }
+
+    refresh_order() {
+        let url = `${this.data.api_url}?get_order=${this.get_url_param('show_order')}&crypto=${this.get_url_param('crypto')}`
```

Comment:
Please see https://github.com/blockonomics/woocommerce-plugin/pull/276/files#diff-32cdce28ac549728f6a28999f6d874aa4f29e6da1406c28c3134cd8d421a866aL183 for forming the woocommerce url: https://github.com/blockonomics/woocommerce-plugin/blob/3302c63a7faa1346cc1faf6d0671e14a0b86e75e/js/app.js#L183-L200